x-pack/plugins/apm/public/selectors/chartSelectors.ts

```diff
@@ -62,6 +63,7 @@ export function getTransactionCharts(
   return {
     tpmSeries,
     responseTimeSeries,
+    mlJobId: anomalyTimeseries?.jobId,
```

Comment:
This doesn't seem like the best place to put `mlJobId` but okay for now.
Would be good to go over chartSelectors and see if we can move it to the backend or something like it.